### ISSUE TITLE
Careportal entries: wrong conversion of mmol BG readings  [BG(mmol)/18]

### ISSUE THREAD

**Issue by bjornoleh** [nightscout/cgm-remote-monitor#5940]
Careportal entries make a wrong conversion of mmol BG readings, where mmol values are reported ad BG(mmol)/18.

Carb correction:
![image](https://user-images.githubusercontent.com/63544115/91877490-5556ae80-ec7e-11ea-89e8-6981c521f269.png)

Meal bolus
![image](https://user-images.githubusercontent.com/63544115/91877766-bd0cf980-ec7e-11ea-91f1-d24646071121.png)

BG Cherck entries are correctly displayed:
![image](https://user-images.githubusercontent.com/63544115/91877551-6f908c80-ec7e-11ea-885a-9ddf4775f9d9.png)

However, this is not consistent across sites. We have four NS sites , two of which are "operational" (set up in April and June 2020). Two additional backup/testing sites were set up recently for migration testing purposes. This bug report is only valid for the sites set up in April and June. The sites are basically set up identically, the only difference is that the two latter sites are not fed with Loop data. The bug has been present since the sites were set up.

The bug is independent of NS Master (0-1 vs Dev. All sites use DISPLAY_UNITS: mmol/L (or mmol, both with the same result).

**Your setup information**
Both Nightscout 13.0.1 and 13.1.0
All sites are fed with Dexcom Bridge CGM data
Same result across browser platforms (Chrome, Safari, Android mobile, iPhone/iPad, PC

**Comment by bjornoleh** [comment]
The Carb Correction entry as stored in Collection: treatments:

`{
    "_id": {
        "$oid": "5f4e7b9ddb69150cd6194bc2"
    },
    "enteredBy": "undefined",
    "eventType": "Carb Correction",
    "glucose": 5,
    "glucoseType": "Finger",
    "carbs": 3,
    "notes": "test BG 5",
    "units": "mmol",
    "created_at": "2020-09-01T16:49:33.868Z",
    "utcOffset": 0
}`

**Comment by sulkaharo** [comment]
This should be fixed in dev

**Comment by bjornoleh** [comment]
Hi, I don't think it is  working in dev as of now, or are you about to change something in dev to fix this?

**Comment by bjornoleh** [comment]
Ok, I just checked with Dev as of today. It looks just like before in the site that was set up in April.

By the way, the marker is positioned correctly at 5 mmol/L in the chart, so it is just the number in the textbox that is wrong (5/18 = 0,3).

![image](https://user-images.githubusercontent.com/63544115/92143049-04c58980-ee15-11ea-81d7-b58fdc70be55.png)

**Comment by bjornoleh** [comment]
Possibly (but not verified) a related observation in CGM In the cloud, but here with mg/dL settings:

https://www.facebook.com/groups/cgminthecloud/permalink/3351883211599578/

**Comment by sulkaharo** [comment]
Does this still replicate with release 14?

**Comment by bjornoleh** [comment]
Yes. There is also still the different behaviour between two NS instances, where the first one give BG (mmol)/18, and the other one displays it correctly.

I also notice now during testing that careportal entries are not updated properly without restarting dynos:

-Entries at the current time are OK
-Entries of past events need restart of dynos
-Deleting careportal entries need restart of dynos

This is different from Dev 13.1.0, but is probably more related to #5992 than the current issue.
